src/main/kotlin/snyk/iac/annotator/IacBaseAnnotator.kt

```diff
@@ -32,11 +36,18 @@ abstract class IacBaseAnnotator : ExternalAnnotator<PsiFile, List<IacIssue>>() {
         LOG.debug("Call apply on ${psiFile.name}")
         if (annotations == null || annotations.isEmpty()) return
 
-        LOG.info("Received ${annotations.size} IacIssue annotations for ${psiFile.virtualFile.name}")
+        LOG.debug("Received ${annotations.size} IacIssue annotations for ${psiFile.virtualFile.name}")
         annotations.forEach { iacIssue ->
             LOG.debug("-> ${iacIssue.id}: ${iacIssue.title}: ${iacIssue.lineNumber}")
 
-            holder.newAnnotation(HighlightSeverity.WARNING, annotationMessage(iacIssue))
+            val severity = when (iacIssue.severity) {
+                CRITICAL -> HighlightSeverity.ERROR
+                HIGH -> HighlightSeverity.ERROR
+                MEDIUM -> HighlightSeverity.WARNING
```

Comment:
we can also use WEAK_WARNING then:
CRITICAL-> ERROR
HIGH -> WARNING
MEDIUM -> WEAK_WARNING 
wdyt?

Reply:
Sounds like a plan :)